ppr-ui/src/composables/fees/FeeSummary.vue

```diff
@@ -178,16 +191,19 @@ export default defineComponent({
   },
   setup (props) {
     const {
-      getLengthTrust, isRoleStaff, getStaffPayment
+      getLengthTrust, isRoleStaff, getStaffPayment, isSearchCertified
     } = useGetters<any>([
-      'getLengthTrust', 'isRoleStaff', 'getStaffPayment'
+      'getLengthTrust', 'isRoleStaff', 'getStaffPayment', 'isSearchCertified'
     ])
     const localState = reactive({
       feeType: props.setFeeType,
       registrationType: props.setRegistrationType,
       hasPriorityFee: computed((): Boolean => {
         return getStaffPayment.value?.isPriority
       }),
+      hasCertifyFee: computed((): Boolean => {
```

Comment:
Could simplify to `hasCertifyFee: computed((): Boolean => isSearchCertified.value)`

Reply:
For sure, I'll clean up the other computed values as well for consistency sake